autoload/fzf/vim.vim

```diff
@@ -634,7 +675,7 @@ function! fzf#vim#gitfiles(args, ...)
   endif
   let prefix = 'git -C ' . fzf#shellescape(root) . ' '
   if a:args != '?'
-    let source = prefix . 'ls-files -z ' . a:args
+    let source = 'git -C ' . fzf#shellescape(root) . ' ls-files -z ' . a:args
```

Comment:
may I ask why we do this? `line 678`
u moved the `diff_prefix` to line 693, and I think this change to `prefix` does not look different from the previous code.

Reply:
I thought `prefix` wasn't used elsewhere, so I decided to remove the variable, but it was a mistake.

Reply:
Yeah that's why I use `diff_prefix` instead of modify `prefix` directly